MathPortal/MathPortal/Source/UserInterface/TabBarViewControllers/CreatedTasks/CreatedTasksViewController.swift

```diff
@@ -17,6 +17,7 @@ class CreatedTasksViewController: BaseViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        tasksTableView?.backgroundColor = UIColor.mathDarkGrey
```

Comment:
Shouldn't really be a part of this PR but would it not be nicer to have like `Colors.darkGrey`? Extending `UIColor` to add constants is usually just a bad idea. You can easily have collisions like the one here having math-dark-grey while you could easily have just dark-grey.